.github/workflows/lint.yaml

```diff
@@ -256,3 +256,38 @@ jobs:
         uses: codespell-project/codespell-problem-matcher@v1
       - name: codespell
         run: ./scripts/spelling.sh codespell
+  weblint:
+    name: weblint
+    runs-on: ubuntu-latest
+    container: debian:stable
+    steps:
+      - name: Update package database
+        run: apt-get update -y
+      - name: Install Git
+        run: apt-get -y install git
+      - uses: actions/checkout@v3
+      - name: Cache NPM
+        uses: actions/cache@v3
+        env:
+          cache-name: cache-npm
+        with:
+          # This is the correct directory to cache
+          # See https://docs.npmjs.com/cli/v8/commands/npm-ci
+          path: ~/.npm
+          key:  debian_stable-lint-weblint-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            debian_stable-lint-weblint-${{ env.cache-name }}-
+            debian_stable-lint-weblint-
+      - name: Setup Node.js APT repo
+        shell: bash
+        run: |
+          apt-get -y install curl
+          curl -fsSL https://deb.nodesource.com/setup_20.x | bash -
+      - name: Install lint tools
+        run: apt-get install -y nodejs
```

Comment:
This can probably be simplified with `actions/setup-node`, see e.g. https://github.com/OpenLightingProject/open-fixture-library/blob/a809729f90eaf635f2aac66ad70bb93c8247c4e1/.github/workflows/test.yaml#L13-L17

Reply:
Fixed!